react-client/src/Components/DomInfoDialogBox.js

```diff
@@ -1,11 +1,23 @@
 import React from "react";
 
-const DomInfoDialogBox = ({ id, idx, clsname, parentId, parentClass, children, top, left, onClose }) => {
-
+const DomInfoDialogBox = ({
```

Comment:
Don't format this one

Reply:
sorry natamaan ng prettier already restored